File: Frontend/src/types/sucursal.ts
export interface Sucursal {
  id: number
  nombre: string
  direccion: string | null
  ciudad: string | null
  telefono: string | null
  activa: boolean
  createdAt: string
}

export interface CrearSucursalRequest {
  nombre: string
  direccion: string | null
  ciudad: string | null
  telefono: string | null
}

export interface ActualizarSucursalRequest extends CrearSucursalRequest {
  activa: boolean
}

File: Backend/Models/Sucursal.cs
namespace InventarioMultiSucursal.Api.Models;

public class Sucursal
{
    public int Id { get; set; }
    public string Nombre { get; set; } = string.Empty;
    public string? Direccion { get; set; }
    public string? Ciudad { get; set; }
    public string? Telefono { get; set; }
    public bool Activa { get; set; } = true;
    public DateTime CreatedAt { get; set; }

    public ICollection<Usuario> Usuarios { get; set; } = new List<Usuario>();
}
